### PR TITLE
Add tutorial for setting up Gitlab CI ios on local machine

### PR DESCRIPTION
**Pull request by ivkean**

### CODE REVIEW

**Comment by tieubao** [comment]
@ivkean In general, we should make it concise and with more screenshots

File: developer-runner.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+The green checkmarks next to the test functions (both in the file, and in the Test navigator) show that all tests passed. We won't be referring to the Xcode project anymore, so if you like, you can close it.
+
+The final step here is to install `xcpretty`. When Xcode builds and tests your project, `xcpretty` will transform the output into something more readable for you.
+
+## Installing and registering the GitLab Runner
+
+Registering runner
+
+```
+$ sudo gitlab-ci-multi-runner register
```
Comment (dattran2346) [inline developer-runner.md]: @ivkean sudo gitlab-runner register, I think?

Comment (tieubao) [inline developer-runner.md]: :+1:

Comment (ivkean) [inline developer-runner.md]: `gitlab-runner` is a deprecated version. we will use `gitlab-ci-multi-runner` instead

Comment (ivkean) [inline developer-runner.md]: @dattran2346 , is this tutorial ok w you and @congthai08

**Comment by congthai08** [comment]
You should sync the statements. Ex: "sudo gitlab-ci-multi-runner register" = "sudo gitlab-runner register"

**Comment by ivkean** [comment]
@congthai08 👍 updated already. Let me know if everything is ok. Then, Han will merge this PR.

/cc @tieubao

**Comment by tieubao** [comment]
LGTM 👍

**Pull request merged**